stl/inc/yvals_core.h

```diff
@@ -1632,6 +1633,7 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 #define __cpp_lib_move_only_function 202110L
 
 #ifdef __cpp_lib_concepts
+#define __cpp_lib_fold                    202207L

```

Comment:
```suggestion
#define __cpp_lib_ranges_fold                    202207L
```
the WP says `__cpp_lib_ranges_fold`, not `__cpp_lib_fold`.